api/src/index.js

```diff
@@ -54,6 +54,10 @@ app.use(versionCheck);
 
 // Pre middleware
 app.use(express.urlencoded({ extended: true }));
+app.use((req, res, next) => {
+  const limitedSize = req.path.startsWith("/encrypt") ? "1gb" : "50mb";
```

Comment:
La base de données en prod total faisait 350 mo en décembre, je pense qu'on peut laisser à moins pour les users mais je suis peut-être nouille.

```suggestion
  const limitedSize = req.path.startsWith("/encrypt") ? "350mb" : "50mb";
```